### PR TITLE
Change remove to another function and rename cleanup #28

### PR DESCRIPTION
**Pull request by kpichardie**

### CODE REVIEW

File: passkeeper/__init__.py [hunk 1]
```diff
@@ -68,7 +68,9 @@ def init_dir(self, passphrase):
             f.write(sample_raw)
 
         self.encrypt(passphrase=passphrase)
-        self.cleanup_ini()
+        # Remove old passkeeper files
+        self.remove_old_encrypted_files(force_remove=False)
```
Comment (gaelL) [inline passkeeper/__init__.py]: `force_remove=False` should be forced to `True`

File: passkeeper/__init__.py [hunk 2]
```diff
@@ -148,7 +150,7 @@ def decrypt(self, passphrase):
         return status
 
 
-    def _remove_old_encrypted_files(self, force_remove=False):
+    def remove_old_encrypted_files(self, force_remove=False):
```
Comment (gaelL) [inline passkeeper/__init__.py]: `force_remove=False` arg is now useless, we can remove it

Comment (gaelL) [inline passkeeper/__init__.py]: `force_remove=False` arg is now useless, we can remove it

File: passkeeper/tests/units/test_passkeeper.py [hunk 3]
```diff
@@ -64,16 +64,17 @@ def test_init_dir(self, mock_create_dir, mock_encrypt, mock_cleanup):
...
     @patch('passkeeper.os.path.isfile')
-    def test_cleanup_ini(self, mock_isfile, mock_listdir, mock_cmd,
-                               mock_remove_old_encrypted_files):
+    def test_cleanup(self, mock_isfile, mock_listdir, mock_cmd,
+                     mock_remove_old_encrypted_files):
         # One ignored file and one valid file
         mock_listdir.return_value = ['ignored', 'bar.ini']
         mock_isfile.return_value = True
-        self.pk.cleanup_ini()
+        self.pk.remove_old_encrypted_files(force_remove=False)
```
Comment (gaelL) [inline passkeeper/tests/units/test_passkeeper.py]: This test is only for cleanup, we don't want to test remove anymore in this test.
Remove is now tester in `test_remove_old_encrypted_files`

File: passkeeper/tests/units/test_passkeeper.py [hunk 4]
```diff
@@ -64,16 +64,17 @@ def test_init_dir(self, mock_create_dir, mock_encrypt, mock_cleanup):
         mock_cleanup.assert_called_once_with()
 
 
-    @patch('passkeeper.Passkeeper._remove_old_encrypted_files')
+    @patch('passkeeper.Passkeeper.remove_old_encrypted_files')
```
Comment (gaelL) [inline passkeeper/tests/units/test_passkeeper.py]: Same as comment below, cleanup don't call remove anymore, no need to mock and test it here.

File: passkeeper/__init__.py (hunk 2 above)
Comment (kpichardie) [inline passkeeper/__init__.py]: Not on the good function, you mean cleanup()

**Comment by shaftmx** [comment]
perfect

**Pull request merged**